sdk/python/feast/infra/online_stores/contrib/postgres.py

```diff
@@ -67,6 +77,26 @@ def _get_conn(self, config: RepoConfig) -> Generator[Connection, Any, Any]:
                 self._conn = _get_conn(config.online_store)
             yield self._conn
 
+    @contextlib.asynccontextmanager
+    async def _get_conn_async(
+        self, config: RepoConfig
+    ) -> AsyncGenerator[AsyncConnection, Any]:
+        if config.online_store.conn_type == ConnectionType.pool:
+            if not self._conn_pool_async:
+                self._conn_pool_async = await _get_connection_pool_async(
+                    config.online_store
+                )
+                await self._conn_pool_async.open()
```

Comment:
Should this `open()` call be here in `_get_conn_async` or in `_get_connection_pool_async`? 🤔